x-pack/plugins/observability/public/pages/alerts/alerts_table_t_grid.tsx

```diff
@@ -0,0 +1,197 @@
+> = [
+  {
+    columnHeaderType: 'not-filtered',
+    displayAsText: i18n.translate('xpack.observability.alertsTGrid.statusColumnDescription', {
+      defaultMessage: 'Status',
+    }),
+    id: ALERT_STATUS,
+    initialWidth: 79,
+  },
+  {
+    columnHeaderType: 'not-filtered',
+    displayAsText: i18n.translate('xpack.observability.alertsTGrid.triggeredColumnDescription', {
+      defaultMessage: 'Triggered',
+    }),
+    id: ALERT_START,
+    initialWidth: 116,
+  },
+  {
+    columnHeaderType: 'not-filtered',
+    displayAsText: i18n.translate('xpack.observability.alertsTGrid.durationColumnDescription', {
+      defaultMessage: 'Duration',
+    }),
+    id: ALERT_DURATION,
+    initialWidth: 116,
+  },
+  {
+    columnHeaderType: 'not-filtered',
+    displayAsText: i18n.translate('xpack.observability.alertsTGrid.severityColumnDescription', {
+      defaultMessage: 'Severity',
+    }),
+    id: ALERT_SEVERITY_LEVEL,
+    initialWidth: 102,
+  },
+  {
+    columnHeaderType: 'not-filtered',
+    displayAsText: i18n.translate('xpack.observability.alertsTGrid.reasonColumnDescription', {
+      defaultMessage: 'Reason',
+    }),
+    linkField: '*',
+    id: RULE_NAME,
+    initialWidth: 400,
+  },
+];
+
+const NO_ROW_RENDER: RowRenderer[] = [];
+
+const trailingControlColumns: never[] = [];
+
+export function AlertsTableTGrid(props: AlertsTableTGridProps) {
+  const { core, observabilityRuleTypeRegistry } = usePluginContext();
+  const { prepend } = core.http.basePath;
+  const { indexName, rangeFrom, rangeTo, kuery, status, setRefetch } = props;
+  const [flyoutAlert, setFlyoutAlert] = useState<TopAlert | undefined>(undefined);
+  const handleFlyoutClose = () => setFlyoutAlert(undefined);
+  const { timelines } = useKibana<{ timelines: TimelinesUIStart }>().services;
+
+  const leadingControlColumns = [
+    {
+      id: 'expand',
+      width: 40,
+      headerCellRender: () => null,
+      rowCellRender: ({ data }: ActionProps) => {
+        const dataFieldEs = data.reduce((acc, d) => ({ ...acc, [d.field]: d.value }), {});
+        const decoratedAlerts = decorateResponse(
+          [dataFieldEs] ?? [],
+          observabilityRuleTypeRegistry
+        );
+        const alert = decoratedAlerts[0];
+        return (
+          <EuiButtonIcon
+            size="s"
+            iconType="expand"
+            color="text"
+            onClick={() => setFlyoutAlert(alert)}
+          />
+        );
+      },
+    },
+    {
+      id: 'view_in_app',
+      width: 40,
+      headerCellRender: () => null,
+      rowCellRender: ({ data }: ActionProps) => {
+        const dataFieldEs = data.reduce((acc, d) => ({ ...acc, [d.field]: d.value }), {});
+        const decoratedAlerts = decorateResponse(
+          [dataFieldEs] ?? [],
+          observabilityRuleTypeRegistry
+        );
+        const alert = decoratedAlerts[0];
+        return (
+          <EuiButtonIcon
+            size="s"
+            target="_blank"
```

Comment:
What's the intention behind opening in a new tab here? Shouldn't that be up to the user?